templates/summit/index.html

```diff
@@ -58,9 +58,12 @@ <h3 class="p-heading--5">The Hague, Netherlands</h3>
         </p>
       </div>
       <div class="col-4 u-vertically-center">
-        <img src="https://assets.ubuntu.com/v1/9e91caf9-Canonical%20Ubuntu%20Summit%20Logo%202024%20with%20text.svg"
-             width="350"
-             alt="Ubuntu Summit 2024 Logo" />
+        {{ image(url="https://assets.ubuntu.com/v1/048df0ca-Canonical%20Ubuntu%20Summit%20Logo%202024%20with%20text.png",
```

Comment:
Why switch to using a png?

Reply:
Check the PR description:

> Converted incompatible images from svg format to png. See https://github.com/canonical/canonicalwebteam.image-template/issues/4

The SVG wasn't rendering properly